extension/src/background/messageListener/popupMessageListener.ts

```diff
@@ -544,6 +544,23 @@ export const popupMessageListener = (request: Request) => {
     }
   };
 
+  const signTransactionXDR = () => {
```

Comment:
open to naming suggestions. Trying to figure the best way to differentiate this method name from the `signTransaction` above, even though they're signing but differently

Reply:
hmmmmm how about something like: `signUserTransactionXDR`?

I think we want it to be clear that Freighter is the one generating this XDR and it's not coming from a client website

I'm gonna create a ticket to change the name of `SignTransaction` to something like `SignClientTransaction` to make it more clear

Reply:
what about `signFreighterTransaction` vs `signClientTransaction` ?

Reply:
I changed it to `signFreighterTransaction`. If you hate it though lmk 🙂
https://github.com/stellar/freighter/pull/372/commits/dd01ed99eb967aabe294f288742bd9c7734b22db